diabetes-v2/usage_config.yml

```diff
@@ -0,0 +1,83 @@
+data_usage:
+  # Define the keys used to identify an output;
+  # entity_id is required while a secondary context grouping is optional
+  entity_id: patient_nbr
+  context_id: encounter_id
+  # Each use case must define a primary output and target
+  # Output should be in the predictions table but target may be a display name of a windowed event
+  primary_output: Risk30DayReadmission
+  # This list defines available outputs for the model (scores)
+  outputs:
+   - Risk30DayReadmission
+   - Risk30DayReadmissionPercentile
+   - RiskAnyReadmission
+   - RiskLongStay
+   - ExpectedHospitalStay
+  # This list defines available outputs for the model (scores)
+  primary_target: Readmitted within 30 Days
+  # Predict time indicates the column for timestamp associated with the row
+  predict_time: ScoringTime
+  # Features, when present, will reduce the data loaded from predictions.
+  # It does NOT need to include cohorts our outputs specified elsewhere
+  features:
+    - admission_type_id
+    - num_medications
+    - num_procedures
+  # This list defines available cohort options for the default selectors
+  cohorts:
+    - source: age
+      display_name: Age
+    - source: race
+      display_name: Race
+    - source: gender
+      display_name: Gender
+    - source: A1Cresult  
+      display_name: A1C
+    - source: insulin  
+      display_name: Taking Insulin
+    - source: metformin   
+      display_name: Taking Metformin
+  # The event_table allows mapping of event columns to those expected by the tool
+  # The table must have the entity_id column and may have context_id column if being used
+  event_table:
+    type: Type
+    time: EventTime
+    value: Value
+  # Events define what types of events to merge into analyses
+  # Windowing defines the range of time prior to the event where predictions are considered
+  events:
+    - source: Readmitted
+      display_name: Readmitted within 30 Days
+      window_hr: 720
+      offset_hr: 0
+      usage: target
+      score_rollup: max
+    - source:  Readmitted
+      display_name: Readmitted (Any)
+      window_hr: 1440
```

Comment:
Can we remove the window+offset from this "Any" target?  
It's nice to demonstrate they aren't required, and arguably no windowing is an expectation for "Any" (though maybe not in terms of rigorous definitions of "hospital readmissions"...)

Reply:
We kind of need the window_hr if we wan to use it the leadtime plot.

Reply:
Ah, that makes sense...